addons/cloud-games/addon.json

```diff
@@ -15,7 +15,64 @@
     "script": "popup.js",
     "html": "popup.html"
   },
+  "settings": [
+    {
+      "id": "displayedGames",
+      "name": "Displayed games",
+      "type": "table",
+      "default": [
+        {
+          "url": "https://scratch.mit.edu/studios/539952/"
+        }
+      ],
+      "row": [
+        {
+          "name": "Project/Studio URL",
+          "id": "url",
+          "type": "untranslated",
+          "default": ""
+        }
+      ],
+      "presets": [
+        {
+          "name": "Griffpatch's Multiplayer Games",
+          "values": {
+            "url": "https://scratch.mit.edu/studios/539952/"
+          }
+        },
+        {
+          "name": "Massive Multiplayer Platformer",
+          "values": {
+            "url": "https://scratch.mit.edu/projects/612229554/"
+          }
+        },
```

Comment:
Griffpatch took down this project :(

Reply:
> MMO has been taken down again due to hackers
https://scratch.mit.edu/users/griffpatch